code/ui/PlayerInfo.cs

```diff
@@ -69,24 +89,44 @@ public override void Tick()
                     return;
                 }
 
-                HealthBar.TextLabel.Text = $"{player.Health:n0}";
+                if (currentHealth != player.Health)
+                {
+                    currentHealth = player.Health;
+
+                    HealthBar.TextLabel.Text = $"{player.Health:n0}";
+
+                    // bar width
```

Comment:
Just another comment that doesn't provide us with any extra info that the code snippet already does (imo)

```
HealthBar.Style.Width = Length.Percent(player.Health);
```